<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	font: 22px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:30px;
	color:#444444;
}</string>
    </property>
    <property name="text">
     <string>实验五：扫雷游戏</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>241</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>模式选择</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>难度：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="level_option">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>简单模式</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>中等模式</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>困难模式</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_confirm">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>选中</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>361</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>扫雷游戏</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="btn_contain"/>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>440</y>
      <width>661</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>控制台</string>
    </property>
    <widget class="QTextBrowser" name="res_console">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>621</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>230</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>游戏记录</string>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>成功概率：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="success_pro">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="best_record">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>最佳成绩：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>剩余雷数：</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="remain_num">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
